spark/sql-20/src/test/scala/org/elasticsearch/spark/sql/SparkSessionTest.scala

```diff
@@ -0,0 +1,26 @@
+package org.elasticsearch.spark.sql
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.SparkSession
+import org.junit.Assert._
+import org.junit.Test
+
+class SparkSessionTest {
+
+  @Test
+  def parameters(): Unit = {
+    val sparkConf = new SparkConf().setAppName("session conf").setMaster("local[*]")
+    val session1 = SparkSession.builder().config(sparkConf).getOrCreate()
+    session1.conf.set("es.nodes", "127.0.0.1")
+    session1.conf.set("es.port", "9200")
+    assertTrue(session1.conf.get("es.nodes").equals("127.0.0.1"))
```

Comment:
I'm not sure that this is testing any of the changes in this PR, maybe it makes sense to do this in the integration tests?

Reply:
The purpose of this test is to show that SparkSession can be configured differently to connect different clusters. This is the problem I have in my work. At work, we need to cache different instances of SparkSession, and use the instance to query different clusters and indexes when hitting cache.

In fact, I also modified the source code, adding two configurations of es.cluster.name and es.index.filter. es.cluster.name is to do the validation of the cluster name, and es.index.filter is to filter out the index I want to query in the large alias (the alias pointing to many indexes). In this way,each SparkSession instance (we cache) can query different indexes through the index of the es.index.filter configuration.